presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1358,16 +1358,17 @@ public HiveClientConfig setS3SelectPushdownMaxConnections(int s3SelectPushdownMa
         return this;
     }
 
-    public boolean isStreamingAggregationEnabled()
+    public boolean isOrderBasedExecutionEnabled()
     {
-        return streamingAggregationEnabled;
+        return orderBasedExecutionEnabled;
     }
 
-    @Config("hive.streaming-aggregation-enabled")
-    @ConfigDescription("Enable streaming aggregation execution")
-    public HiveClientConfig setStreamingAggregationEnabled(boolean streamingAggregationEnabled)
+    @Config("hive.order-based-execution-enabled")
```

Comment:
Anyone reasonably expected to be using this config?  In which case, we'll want to deprecate the old naming.

Reply:
This will be shared by both the streaming aggregation and merge join feature to expose the ordering property of the table and disable file splitting.
The original naming is confusing if we also use it for merge join.